scripts/parse_lake_manifest_changes.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+
+import json
+import subprocess
+import sys
+
+def get_git_diff():
+    """Get the git diff for lake-manifest.json between HEAD and HEAD~1"""
+    try:
+        result = subprocess.run(
+            ['git', 'diff', 'HEAD~1..HEAD', '--', 'lake-manifest.json'],
+            capture_output=True,
+            text=True,
+            check=True
+        )
+        return result.stdout
+    except subprocess.CalledProcessError:
+        print("Error: Failed to get git diff", file=sys.stderr)
+        return None
+
+def parse_manifest_changes(diff_text):
+    """Parse the diff to find version changes in dependencies"""
+    changes = []
+    old_version = None
+    new_version = None
+    package_name = None
+
+    lines = diff_text.split('\n')
+    for i, line in enumerate(lines):
+        # When we find a rev change
+        if line.startswith('-   "rev":'):
```

Comment:
This seems very fragile; let's parse the json instead and compare without a diff tool?